app/lib/ui/components/cards/card_small.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:ods_flutter/components/card/ods_small_card.dart';
+import 'package:ods_flutter/guidelines/spacings.dart';
+import 'package:ods_flutter_demo/constants.dart';
+
+class CardSmall extends StatefulWidget {
+  const CardSmall({super.key});
+
+  @override
+  State<CardSmall> createState() => _CardSmallState();
+}
+
+class _CardSmallState extends State<CardSmall> {
+  @override
+  Widget build(BuildContext context) {
+    var cardWidth = -2 * spacingM;
+    if (MediaQuery.of(context).size.width < mobileUiMaxScreenWidth) {
+      cardWidth += MediaQuery.of(context).size.width / 2;
+    } else {
+      cardWidth += MediaQuery.of(context).size.width / 3.5;
+    }
+
+    return Scaffold(
+      backgroundColor: Colors.white,
+      appBar: AppBar(
+        title: Text('Small Card'),
+      ),
+      body: Padding(
+        padding: const EdgeInsets.all(spacingM),
+        child: Column(
+          children: <Widget>[
```

Comment:
no need <Widget> here

Reply:
If I remove the column, the card takes full screen height. 
I don't find any other solution: https://stackoverflow.com/questions/42257668/the-equivalent-of-wrap-content-and-match-parent-in-flutter